find_me_a_sandwich/spec/models/favorites_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+describe Favorite, type: :model do
+  let(:favorite) { FactoryGirl.build(:favorite) }
+  let(:users) do
+    FactoryGirl.create_list(:named_user,
+      10,
+      profile_visible: true)
+  end
+  let(:_) do
```

Comment:
If you want to run this without assigning to a variable, just use `before(:each)`